.phpcs.xml.dist

```diff
@@ -3,6 +3,8 @@
 
 	<rule ref="WebDevStudios"/>
```

Comment:
I wonder if you should use WordPress is doing here too? I'd also like to "reboot" PHP Coding Standards to match WordPress as much as possible and e.g. configure things like short-array-syntax, etc either at the config level (here) —especially if it's just this ONE config— or create a new package that has e.g. this tweak that you can install.

Reply:
@aubreypwd Sounds good. I'll remove it and sprinkle in short-array-syntax.